### ISSUE TITLE
Realchute stack "hovers" above other parts.

### ISSUE THREAD

**Issue by pjf** [KSP-RO/RealismOverhaul#63]
Actual behaviour: The Realchute stack (`RC_stack` in `RealismOverhaul/RO_RealChute.cfg`) has top and bottom nodes which aren't on the surface of the compoent itself, causing it to 'hover' when placed in an actual stack.

Expected behaviour: Attachment nodes are on the surface of the part itself.

Steps to reproduce: Select the RC_stack part in the VAB and attach it to a part. (I'm using just an okto2 and the stack chute, and there's a very noticable gap.)

Note: I haven't tried RealChute without RO, so this could be stock behaviour.

Mod loadout:

```
KSP found at /home/pjf/.steam/steam/SteamApps/common/Kerbal Space Program

KSP Version: 0.25.0

Installed Modules:

✓ AGExt 1.24a
✓ AJE 1.6.4
✓ AlternateResourcePanel 2.6.1.0
✓ Chatterer 0.7.1
✓ CIT-Util 1.0.4-unofficial
✓ CommunityResourcePack 0.2.3
✓ CommunityTechTree 1.1
✓ CrossFeedEnabler v3.1
✓ CustomBiomes 1.6.8
✓ CustomBiomes-Data-RSS v8.3
✓ DDSLoader 1.7.0.0
✓ DeadlyReentry v6.2.1
✓ DMagicOrbitalScience 0.9.0.2
✓ DogeCoinFlag 1.02
✓ EngineIgnitor-Unofficial-Repack 3.4.1.1
✓ EVE-Overhaul-Core 0.0.2014.11.25
✓ FerramAerospaceResearch v0.14.4
✓ FinalFrontier 0.5.9-177
✓ FinePrint 0.59
✓ FirespitterCore 7.0.5398.27328
✓ HotRockets 7.25
✓ InfernalRobotics 0.19.2
✓ Karbonite 0.4.4
✓ KAS 0.4.9
✓ KerbalAlarmClock v3.0.5.0
✓ KerbalConstructionTime 1.0.3
✓ KerbalJointReinforcement v2.4.4
✓ kOS 0.15.3.0
✓ KronalVesselViewer 0.0.4_0.25
✓ MechJeb2 2.4.0
✓ ModuleManager 2.5.1
✓ ModuleRCSFX v3.3
✓ NathanKell-RVE-Haxx 0.0.2014.11.25
✓ ORSX 0.1.3
✓ PartCatalog 3.0_RC8
✓ PlanetShine 0.2.2
✓ PreciseNode 1.1.1
✓ ProceduralDynamics 0.9.1
✓ ProceduralFairings v3.10
✓ ProceduralParts v0.9.20
✓ RealChute 1.2.6
✓ RealFuels rf-v8.1-really-v8.2-pre
✓ RealismOverhaul v7.0.2
✓ RealSolarSystem v8.3
✓ RemoteTech v1.5.1
✓ RemoteTech-Config-RSS 0.0
✓ RP-0 v0.13
✓ RSSTextures4096 DDS-1.0
✓ SCANsat v8.0
✓ ScienceAlert 1.8rc1
✓ Service-Compartments-6S 1.2
✓ ShipManifest 0.25.0_3.3.2b
✓ StageRecovery 1.5.1
✓ SXT 18.6
✓ TACLS v0.10.1
✓ TACLS-Config-RealismOverhaul v7.0.2
✓ TechManager 1.5
✓ Toolbar 1.7.7
↑ TweakScale v1.44
✓ UKS 0.21.3
✓ UniversalStorage 0.9.4
✓ UniversalStorage-KAS 0.9.0.14
✓ UniversalStorage-TAC 0.9.2.7
✓ USITools 0.2.4

Legend: ✓ - Up to date. ✗ - Incompatible. ↑ - Upgradable. ? - Unknown 
```

**Comment by RedAV8R** [comment]
Fixed when #556 is merged. @NathanKell

**Comment by Phredward** [comment]
Awesome!  Thanks for fixing this issue!
